### ISSUE TITLE
Windows 10 versions aren't detected/decoded properly:

### ISSUE THREAD

**Issue by marchello2000** [bugsnag/bugsnag-dotnet#44]
For apps that define Win10 support in their manifest:

```
  <compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1">
    <application>
      <!-- Windows 10 -->
      <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
    </application>
  </compatibility>
```

Bugsnag will report `UNKNOWN` version
https://github.com/bugsnag/bugsnag-dotnet/blob/master/src/Bugsnag/Diagnostics.cs#L150

**Comment by CodeHex** [comment]
Addressed in #46, but will only work if Win10 support is added to manifest (see #45)

**Comment by CodeHex** [comment]
Closing this as it has been partially fixed and remaining problem is documented in #45
